src/main/java/Parser.java

```diff
@@ -0,0 +1,66 @@
+import duke.DukeException;
+import java.util.Scanner;
+
+public class Parser {
+    //private TaskList tasks;
+
+    public Parser() {
+        //tasks = new TaskList();
+    }
+    public static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+    public static void handleCommands() {
+        Scanner in = new Scanner(System.in);
+        String command = in.nextLine();
+
+        while (!(command.equals("bye"))) {
+            try {
+                String[] input = command.split(" ", 2); //only check the first word
+                String firstWord = input[0];
+                switch (firstWord) {
+                    //list tasks
```

Comment:
Since this part of the code is self-explanatory, you don't need to comment here. Similarly for the other case statements. [See here](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#do-not-repeat-the-obvious)